shell/src/main/java/alluxio/cli/LogLevel.java

```diff
@@ -0,0 +1,242 @@
+ */
+
+package alluxio.cli;
+
+import alluxio.Constants;
+import alluxio.client.block.AlluxioBlockStore;
+import alluxio.client.block.BlockWorkerInfo;
+import alluxio.util.network.HttpUtils;
+import alluxio.util.network.NetworkAddressUtils;
+import alluxio.util.network.NetworkAddressUtils.ServiceType;
+import alluxio.wire.LogInfo;
+import alluxio.wire.WorkerNetAddress;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.apache.commons.lang.StringUtils;
+import org.apache.http.client.utils.URIBuilder;
+
+import javax.annotation.concurrent.NotThreadSafe;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Set and get the specify target specify log's level.
+ */
+@NotThreadSafe
+public final class LogLevel {
+
+  private static final String LOG_LEVEL = "logLevel";
+  private static final String TARGET_OPTION_NAME = "target";
+  private static final Option TATGET_OPTION =
+      Option.builder().required(false).longOpt(TARGET_OPTION_NAME).hasArg(true)
+          .desc("<master|workers|host:port>."
+              + " Multi target split by ',' host:port pair must be one of workers."
+              + " Default target is master and all workers").build();
+  private static final String LOG_NAME_OPTION_NAME = "logName";
+  private static final Option LOG_NAME_OPTION =
+      Option.builder().required(true).longOpt(LOG_NAME_OPTION_NAME).hasArg(true)
+          .desc("The log name you want to get or set level.").build();
+  private static final String LEVEL_OPTION_NAME = "level";
+  private static final Option LEVEL_OPTION =
+      Option.builder().required(false).longOpt(LEVEL_OPTION_NAME).hasArg(true)
+          .desc("The level of the log what you specified.").build();
+  private static final Options OPTIONS = new Options()
+      .addOption(TATGET_OPTION)
+      .addOption(LOG_NAME_OPTION)
+      .addOption(LEVEL_OPTION);
+
+  private static final String USAGE =
+      "logLevel --logName=LOGNAME [--target=<master|workers|host:port>] [--level=LEVEL]";
+  public static final String TARGET_SEPARATOR = ",";
+
+  /**
+   * Prints the help message.
+   *
+   * @param message message before standard usage information
+   */
+  public static void printHelp(String message) {
+    System.err.println(message);
+    HelpFormatter help = new HelpFormatter();
+    help.printHelp(LOG_LEVEL, OPTIONS, true);
+  }
+
+  /**
+   * Implements log level setting and getting.
+   *
+   * @param args list of arguments contains target, logName and level
+   * @return 0 on success, 1 on failures
```

Comment:
try to catch the exception in `main`, set the return value of the command in `main`